object/src/mem.rs

```diff
@@ -889,7 +907,7 @@ where
                 AttributeSelectorStep::Nested { tag, item } => {
                     let e = obj
                         .entries
-                        .get(tag)
+                        .get(&tag)
```

Comment:
This is just a false positive from rust-analyzer. https://github.com/rust-lang/rust-analyzer/issues/16657

```suggestion
                        .get(tag)
```